transfer-products/src/transfer_products.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import os
+from dataclasses import dataclass
+
+import boto3
+import boto3.s3.transfer
+import botocore.exceptions
+import hyp3_sdk
+import requests
+from dotenv import load_dotenv
+
+S3 = boto3.resource('s3')
+
+TARGET_BUCKET = 'hyp3-nasa-disasters'
```

Comment:
this might be better extracted to a cloudformation stack parameter, especially since we'll need to grant the lambda permissions to write to the target bucket